endaq/calc/utils.py

```diff
@@ -14,7 +14,7 @@
 def sample_spacing(
     data: Union[np.ndarray, pd.DataFrame],
     convert: typing.Literal[None, "to_seconds"] = "to_seconds",
-) -> Union[float, np.timedelta64]:
+) -> Optional[Union[float, np.timedelta64]]:
```

Comment:
`Optional` is intended as syntactic sugar for parameters; it would be better to explicitly state the output (`Union[None, float, np.timedelta64]`).

Reply:
sure. see 6266a9cc63546729f4ba187e289f6e5e3f8a7e02